src/schemas/json/project.json

```diff
@@ -75,7 +76,8 @@
 					  "enum": [ "project", "package" ]
 					}
 				}
-			}
+			},
+   			"description": "The dependencies of the application. Each is defined by name and version. The runtime loaders will determine what should be loaded: NuGet package, source code, etc."
```

Comment:
When moving to dotnet-cli, there isn't really a "runtime loader" any more. I'd rephrase more as 

> Each dependency is defined by a name and a version. Dependencies are resolved from NuGet feeds defined by your package sources and Projects located in the directories specified by the 'global.json' file"